.github/workflows/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+Thanks a lot for contributing to skops! Please make sure you've taken a look at the contribution guidelines: https://github.com/skops-dev/skops/blob/main/CONTRIBUTING.rst You can find answers to your questions about the checklist below in this file.
+Below is a template you can use to create pull requests. 
+
+# Description
+
+Please include:
+* relevant motivation
+* a summary of the change
+* which issue is fixed
+* any additional dependencies that are required for this change.
+
+You can link the PR to the issue like below:
+Closes: # (issue)
+If it's a PR that solves a small part of the issue, you can do:
+Partially resolves: # (issue)
+
+# Checklist:
+
+- [ ] I have performed a self-review of my own code
+- [ ] I have run style formatting on my code
+- [ ] I have commented my code in hard-to-understand areas
+- [ ] I have made changes to docstrings and notebooks
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] I added my changes to changelog in docs/changes.rst.
+
+Don't hesitate to ping @skops-dev/maintainers in your issues and pull requests if you don't receive a review in a timely manner. We try to review all pull requests as soon as we can.
```

Comment:
I don't think this is useful. It sounds a bit condescending to me.

Reply:
"I performed a self review of my own code"? That's really bad. Also, not ever PR needs a change log. Also, we send PRs w/o the tests passing all the time, and that's fine, we fix them on the PR.